dockers/docker-lldp-sv2/lldpmgrd

```diff
@@ -185,37 +189,52 @@ class LldpManager(object):
     def run(self):
         """
         Subscribes to notifications of changes in the PORT table
-        of the Redis State database. 
-        Subscribe to STATE_DB - get notified of the creation of an interface
+        of the Redis Config/Application database.
+        Subscribe to APP_DB - get port oper status
         Subscribe to CONFIG_DB - get notified of port config changes
         Update LLDP configuration accordingly.
         """
         # Set select timeout to 10 seconds
         SELECT_TIMEOUT_MS = 1000 * 10
 
-        # Subscribe to PORT table notifications in the State DB
         sel = swsscommon.Select()
-        sst = swsscommon.SubscriberStateTable(self.state_db, swsscommon.STATE_PORT_TABLE_NAME)
-        sel.addSelectable(sst)
-        sst = swsscommon.SubscriberStateTable(self.config_db, swsscommon.CFG_PORT_TABLE_NAME)
-        sel.addSelectable(sst)
 
-        # Listen for changes to the PORT table in the STATE_DB and CONFIG_DB
+        # Subscribe to PORT table notifications in the Config DB
+        sst_confdb = swsscommon.SubscriberStateTable(self.config_db, swsscommon.CFG_PORT_TABLE_NAME)
+        sel.addSelectable(sst_confdb)
+
+        # Subscribe to PORT table notifications in the App DB
+        sst_appdb = swsscommon.SubscriberStateTable(self.appl_db, swsscommon.APP_PORT_TABLE_NAME)
+        sel.addSelectable(sst_appdb)
+
+        # Listen for changes to the PORT table in the CONFIG_DB and APP_DB
         while True:
+            state = None
```

Comment:
We initialize state variable later, overwriting current None. So I think it's better to delete this line.

Reply:
Will remove it

---
In reply to: [303182812](https://github.com/Azure/sonic-buildimage/pull/3152#discussion_r303182812) [](ancestors = 303182812)